app/Resources/views/full/contact_form.html.twig

```diff
@@ -39,24 +39,20 @@
                             }) }}
                         </div>
                         <div class="form-group">
-                            {{ form_label(form.body, 'Anything you want to add:'|trans, {
+                            {{ form_label(form.message, 'Anything you want to add:'|trans, {
                                 label_attr: {
                                     class: 'text-about'
                                 }
                             }) }}
-                            {{ form_widget(form.body, {
+                            {{ form_widget(form.message, {
                                 attr: {
                                     placeholder: 'Your message'|trans,
                                     class: 'form-control'
                                 }
                             }) }}
                         </div>
-                        {{ form_widget(form.submit, {
-                            attr: {
-                                class: 'btn btn-primary btn-lg'
-                            },
-                            label: 'Send'|trans
-                        }) }}
+                        <button type="submit"  class="btn btn-primary btn-lg">{{ 'Send'|trans }}</button>
```

Comment:
CS: there is a small extra space between `"submit"` and `class...` which you can remove ;)